helpers/helpers.js

```diff
@@ -95,5 +95,18 @@ module.exports = {
 		}
 
 		return ret;
+	},
+	jekyllFrontMatter: function(context, options) {
```

Comment:
Yeah. it uses tabs (4). I didn't want to change it, yet. At least not in this PR.